vertx-web-api-service/src/main/asciidoc/index.adoc

```diff
@@ -183,3 +188,38 @@ Now you can register your service to event bus:
 ----
 
 For more info on how to expose your service look at https://vertx.io/docs/vertx-service-proxy/java/#_exposing_your_service[Vert.x service proxy documentation]
+
+== OpenAPI Integration
+
+Vert.x Web API Service offers an {@link io.vertx.ext.web.api.service.OpenAPIRouterHandler} which can be added to an {@link io.vertx.ext.web.openapi.router.OpenAPIRoute} and will forward incoming HTTP requests to the related Web API Service Proxy.
+The validation of this {@link io.vertx.ext.web.openapi.router.OpenAPIRoute} *MUST* be enabled.
+
+[source,$lang]
+----
+{@link examples.OpenAPIExamples#addHandler}
+----
+
+=== OpenAPI contract extension
+
+Adding an {@link io.vertx.ext.web.api.service.OpenAPIRouterHandler} to an {@link io.vertx.ext.web.openapi.router.OpenAPIRoute} requires to extend the related `OpenAPI Operation` with a `x-vertx-event-bus` extension.
+The `x-vertx-event-bus` extension must be filled with the address and method name of the related Web API Service Proxy.
+If no method name is defined, the operationId will be used.
+
+[source,yaml]
+----
+# without method name
+paths:
+  /pets:
+    get:
+      operationId: listPets
+      x-vertx-event-bus: myEventbusAddress
+
+# with method name
+paths:
+  /pets:
```

Comment:
in the old module, it was possible to put the `x-vertx-event-bus` key under the path directly and mount all the operations this way, will this be also possible here?

Reply:
At the moment the "Operation" is passed into the handler, which has no access to the OpenAPI Model of its path.

We would need to parse the extensions and add them to all operations related to the path. Like we do it for parameters [1].

[1] https://github.com/eclipse-vertx/vertx-openapi/blob/d0be35dbf9df2f69d69baef0b5e5e32b333e0b66/src/main/java/io/vertx/openapi/contract/Path.java#L42

Reply:
maybe its a good idea in general for the Operation to keep a reference to its Path as I can think of may use cases there one would need to find the related path of an operation. It will then also indirectly solve this problem without needing to add exceptions to how specification is mapped to mode. right?

Reply:
it is now possible

Reply:
thanks for going the extra mile. would you update the tests and docs mentioning this too?

Reply:
Good point, I added an example